pyproject.toml

```diff
@@ -10,31 +10,32 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.6.*"
-authutils = "^4.0.0"
+python = "^3.6.1"
+authutils = "^6.0.0"
 cdislogging = "^1.0.0"
-cdiserrors =  "^0.1.2"
+cdiserrors =  "^1.0.0"
 doiclient = {git = "https://github.com/uc-cdis/doiclient", rev = "1.0.0"}
 dosclient = {git = "https://github.com/uc-cdis/dosclient", rev = "1.1.0"}
 gen3authz = "^1.0.4"
 hsclient = {git = "https://github.com/uc-cdis/hsclient", rev = "1.0.0"}
 indexclient = "^2.1.0"
-jsonschema = "^2.5.1"
-flask = "^1.1.1"
+jsonschema = "^3.2"
+flask = "^2.0.1"
+pandas= "^1.0"
```

Comment:
why do we need pandas?

Reply:
because of cython, which we needed for... I can't remember but I can try and recreate if you want

Reply:
that's fine but could you add a comment on why we need it 😬 i know it's annoying to wait for jenkins again but context is nice when we update these files - maybe use a label to limit jenkins to a small set of tests to speed it up

Reply:
in the toml right? 😳

Reply:
yes